dist/index.js

```diff
@@ -69901,13 +69901,28 @@ async function ScanPullRequest(
     for (let i = 0; i < sarif.runs[0].results.length; i++) {
       let results = sarif.runs[0].results[i];
       const commit_sha = results.partialFingerprints.commitSha;
+      const fingerprint =
+        commit_sha +
+        ":" +
+        results.locations[0].physicalLocation.artifactLocation.uri +
+        ":" +
+        results.ruleId +
+        ":" +
+        results.locations[0].physicalLocation.region.startLine;
 
       let proposedComment = {
         owner: owner,
         repo: repo,
         pull_number: eventJSON.number,
-        body: `🛑 **gitleaks** has detected a secret with rule-id \`${results.ruleId}\` in commit ${commit_sha}
-If this secret is a true positive, please rotate the secret ASAP.`,
+        body: `🛑 **Gitleaks** has detected a secret with rule-id \`${results.ruleId}\` in commit ${commit_sha}.
```

Comment:
Should the commit sha be wrapped in backticks like this? "in commit \`${commit_sha}\`"